condor_watch_q.py

```diff
@@ -512,8 +511,13 @@ def row_data_from_job_state(clusters):
                 active_job_ids.append("{}.{}".format(cluster.cluster_id, proc_id))
 
     row_data[TOTAL] = sum(row_data.values())
-    row_data[ACTIVE_JOBS] = ", ".join(active_job_ids)
+    
+    if len(active_job_ids) > 3:
+        active_job_ids = [active_job_ids[0], active_job_ids[-1]]
+        row_data[ACTIVE_JOBS] = " ... ".join(active_job_ids)
+        return row_data
 
+    row_data[ACTIVE_JOBS] = ", ".join(active_job_ids)
     return row_data
```

Comment:
I'd prefer a single `return` here - can you shift the alternate `row_data[ACTIVE_JOBS]` assignment into a an `else`?